docs/web/about/index.md

```diff
@@ -0,0 +1,23 @@
+# Ibis: Python Data Analysis Productivity Framework
+
+Ibis is a toolbox to bridge the gap between local Python environments (like
+pandas and scikit-learn) and remote storage and execution systems like Hadoop
+components (like HDFS, Impala, Hive, Spark) and SQL databases (Postgres,
+etc.). Its goal is to simplify analytical workflows and make you more
+productive.
+
+We have a handful of specific priority focus areas:
+
+- Enable data analysts to translate local, single-node data idioms to scalable
+  computation representations (e.g. SQL or Spark)
+- Integration with pandas and other Python data ecosystem components
+- Provide high level analytics APIs and workflow tools to enhance productivity
+  and streamline common or tedious tasks.
+- Integration with community standard data formats (e.g. Parquet and Avro)
+- Abstract away database-specific SQL differences
+
+As the [Apache Arrow](http://arrow.apache.org/) project develops, we will
```

Comment:
This paragraph confuses me a bit. It seems to be suggesting a future direction that somehow uses arrow to help execute in Spark and Impala?

Reply:
This page is a copy of the beginning of the current documentation index.

Happy to update it here if it makes sense. I don't really know if it's still relevant or not.

Reply:
Ah cool, I thought you just wrote this so was hoping you had more insight. No worries, can always be changed later.